### ISSUE TITLE
Random abandon mid game

### ISSUE THREAD

**Issue by cortereal00** [ValveSoftware/Dota2-Gameplay#14988]
### Description

I encountered a bug during a ranked game as Lion in Dota 2. Around 50 minutes, I noticed an abandon due to AFK message that appeared in the chat. I was actively playing throughout the entire match, both before and after the message appeared. I believe I got this message because I haven't got any xp for 5 minutes. Anything I can do to get my MMR back?

### Example Match ID (and possibly Timestamp)

7539307332

### Screenshots

![image](https://github.com/ValveSoftware/Dota2-Gameplay/assets/46359605/bcf9917b-157d-44ac-b8b5-c12fdb357f08)
![image](https://github.com/ValveSoftware/Dota2-Gameplay/assets/46359605/982ad5ab-f740-46a0-9179-37050e2f0d65)
![image](https://github.com/ValveSoftware/Dota2-Gameplay/assets/46359605/28f02919-a8a1-4f39-9e9d-54a6a8a2623f)

**Comment by TobyFlendersonFromHR** [comment]
yeah, you can play the game like a normal person instead of running around doing nothing for 5 minutes straight like a bot, idiot.
